specification/ai/ModelClient/models/chat_completions.tsp

```diff
@@ -626,17 +631,33 @@ union ChatMessageImageDetailLevel {
   high: "high",
 }
 
-@doc("A structured chat content item containing an audio content.")
-model ChatMessageAudioContentItem extends ChatMessageContentItem {
+@doc("A structured chat content item for audio content passed as a url pointer.")
+model ChatMessageAudioUrlContentItem extends ChatMessageContentItem {
+  @doc("The discriminated object type: always 'audio_url' for this type.")
+  type: "audio_url";
+
+  @doc("The details of the audio url pointer.")
```

Comment:
Not sure "url pointer" is a term I would use here. I would remove "pointer". And elsewhere where you have this term.